src/aac/templates/genplug/README.md.jinja2

```diff
@@ -0,0 +1,34 @@
+# {{plugin.name}}
+
+## Plugin Commands
+
+### Shell Command
+
+```bash
+(.env) $ aac {{plugin.name}}
```

Comment:
Does this assume the plugin name is the same as the command name?  A plugin (ex: aac-puml) may have multiple commands, none of which match the plugin name.

Reply:
Resolved in 1f38927